docs/README.trae.md

```
@@ -0,0 +1,71 @@
+# Superpowers for TRAE
+
+Guide for using Superpowers with TRAE via native skill discovery.
+
+## Quick Install
+
+Tell TRAE:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)